### DISCUSSION TITLE
Add SlimeFun Support

### DISCUSSION THREAD

**Discussion by FinestAF** [GriefPrevention/GriefPrevention#1806] [Ideas]
Add support so that slimefun machines and tools are protected in claims.

**Comment by bigpresh** [comment]
I'd say in general, GP should not have loads of special-cases to support other plugins - it reacts to Bukkit events, so if another plugin fires Bukkit events when players do things, then GP should handle that just fine with no need for specific support for that plugin.

If there's extra stuff that needs to be handled differently, that's probably a good place for a GP addon, rather than being in GP's core, IMO.
